website/addons/osfstorage/templates/log_templates.mako

```diff
@@ -1,6 +1,7 @@
 <script type="text/html" id="osf_storage_file_added">
 added file
-<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">{{ params.path }}</a> in
+<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">
+    {{ params.path.startsWith('/') ? params.path.replace('/', '', 1) : params.path }}</a> in
```

Comment:
`String.startsWith` is not supported on all browsers, e.g. Safari. Probably better to use `String.replace`, e.g.

```js
params.path.replace(/^\/+/, '')
```